assets/app/view/spreadsheet.rb

```diff
@@ -96,10 +97,17 @@ def render_title
     end
 
     def render_corporations
-      @game.corporations.map { |c| render_corporation(c) }
+      current_round = [@game.turn]
```

Comment:
ok let's refactor this a bit

add a new method to game/base.rb

`
def turn_round_num #or better name
   [@turn, @game.round.round_num] # make round/base return round_num 1 by default
end
 `

Reply:
in corporation.rb#add_operating_info, use that method as well

Reply:
Hmm, are you sure about it being in base? I ask because `round_num` is undefined for non-operating rounds :p.

Reply:
Ok nevermind my comment - I will default it to 1 if it does not exist!